C/Coin-OR/Clp/build_tarballs.jl

```diff
@@ -45,19 +55,25 @@ fi
     --disable-dependency-tracking \
     --enable-shared \
     lt_cv_deplibs_check_method=pass_all \
-    --with-blas="-lopenblas" \
-    --with-lapack="-lopenblas" \
+    --with-blas \
+    --with-blas-lib="-L${libdir} ${LBT}" \
+    --with-lapack \
+    --with-lapack-lib="-L${libdir} ${LBT}" \
+    --with-coinutils \
     --with-coinutils-lib="-lCoinUtils" \
-    --with-osi-lib="-lOsi -lCoinUtils" \
-    --with-mumps-lib="-L${libdir} -ldmumps -lzmumps -lcmumps -lsmumps -lmumps_common -lmpiseq -lpord -lmetis -lopenblas -lgfortran -lpthread" \
-    --with-mumps-incdir="${includedir}/mumps_seq" \
-    --with-metis-lib="-L${libdir} -lmetis" \
-    --with-metis-incdir="${includedir}"
+    --with-osi \
+    --with-osi-lib="-lOsi" \
+    --with-mumps \
+    --with-mumps-lib="-L${libdir} -ldmumps" \
+    --with-mumps-cflags="${includedir}/mumps_seq"
 
 make -j${nproc}
 make install
 """
 
+platforms = supported_platforms()
```

Comment:
```suggestion
# platforms is a global constant defined in coin-or-common.jl
```

Reply:
```
# These are the platforms we will build for by default, unless further
# platforms are passed in on the command line
platforms = expand_cxxstring_abis(supported_platforms())
platforms = filter!(!Sys.isfreebsd, platforms)
```
We don't want to use `expand_cxxstring_abis` is the code if it is not in C++.
 We don't apply `expand_gfortran_versions` by default for the same reason.
 I also don't think that FreeBSD is an issue now.

I suppose that you added these options a long time ago.

Reply:
> We don't want to use expand_cxxstring_abis is the code if it is not in C++

Clp is written in C++?